pkg/utils/fs.go

```diff
@@ -222,3 +222,66 @@ func CalcFileChecksum(fs v1.FS, fileName string) (string, error) {
 
 	return fmt.Sprintf("%x", h.Sum(nil)), nil
 }
+
+// CopyDir copies the contents of the source directory to the destination directory.
+func CopyDir(src string, dst string) error {
```

Comment:
should this be part of the sdk?

Reply:
or the standard golang library so people stops reimplementing it every week

Reply:
LOL that's so true